hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/Edge.java

```diff
@@ -676,7 +692,12 @@ public int getClassId() {
         /**
          * This policy sends each item to all candidate processors.
          */
-        BROADCAST
+        BROADCAST,
+        /**
+         * This policy sends each item to a single processor on each of the
```

Comment:
This doesn't state which processor on each member. The PR mentions "round robin" so I guess it's a different processor every time?

Reply:
Slightly reworded.

Reply:
I think it would be better to describe the procedure made in a round-robin fashion in another sentence. Because it's a bit unclear what exactly is done in a round-robin fashion.

Reply:
Let's follow the lead of UNICAST, which says "with no restriction on the choice." Here we can say "sends each item to a single processor on every cluster member. Which exactly processor inside a member it chooses is arbitrary and may change with every item.".

Reply:
Currently, we can only run a DAG containing a fanout edge in a cluster containing multiple members. Is this expected behavior? I would expect it to act like unicast when it is running on single member because we can normally run a DAG with a distributed edge on a single member.
Otherwise, failing the job in this case is the expected behavior, let's note it here. Also, I think that we should add tests to RoutingPolicyTest for distributed edge types to demonstrate the single member behavior of them (I am aware that currently other distributed edge types isn't tested for their single member behaviors either.)